resources/management/commands/import_berths_from_json.py

```diff
@@ -94,6 +94,10 @@
 
 class Command(BaseCommand):
     def __init__(self, *args, **kwargs):
+        raise DeprecationWarning(
+            "This command imports berth dimensions in centimeters / integers, when the model uses now meters / "
```

Comment:
tiniest nitpicks possible, but my OCD eyes hurt a bit 👀 :
1. could the 2 lines be split evenly
2. could the trailing space on the second line be avoided:
```python
raise DeprecationWarning(
    "This command imports berth dimensions in centimeters / integers, "
    "when  the model uses now meters / decimals. Update it before using."
)
```

Reply:
😂 Fixed.